app.py

```diff
@@ -19,26 +18,49 @@ def index():
     return {"message": data}
 
 
-
 @app.route('/recommend/popular')
 def popular_movies():
     data = recommend_popular_movies()
     return data
 
 
```

Comment:
These two lines are not indented properly. They should be inside a function or route decorator.